#
# ROSMIP Controller Debug
#
float64 theta
float64 phiL
float64 phiR
float64 thetad
float64 phidL
float64 phidR

float64 theta_sp
float64 phiL_sp
float64 phiR_sp

float64 tauL
float64 tauR

